deploy/operator/ztp/deploy_spoke_cluster.sh

```diff
@@ -26,6 +26,7 @@ export SPAWN_NONE_PLATFORM_LOAD_BALANCER="${SPAWN_NONE_PLATFORM_LOAD_BALANCER:-f
 export ADD_NONE_PLATFORM_LIBVIRT_DNS="${ADD_NONE_PLATFORM_LIBVIRT_DNS:-false}"
 export LIBVIRT_NONE_PLATFORM_NETWORK="${LIBVIRT_NONE_PLATFORM_NETWORK:-ostestbm}"
 export LOAD_BALANCER_IP="${LOAD_BALANCER_IP:-192.168.111.1}"
+export API_IP=LOAD_BALANCER_IP
```

Comment:
This will create a variable called "API_IP" container the literal string `"LOAD_BALANCER_IP"`